index.ts

```diff
@@ -35,24 +41,24 @@ async function getUrls(configFileName: string): Promise<string[]> {
   process.exit(0);
 }
 
-async function getParser(
-  parserFileName: string
-): Promise<(arg0: Response) => void> {
-  const parser = await import(`${PARSER_PATH}/${parserFileName}`);
-  if (!parser.default) {
+async function getProcessor(
+  processorFileName: string
+): Promise<(arg0: Response) => void | Promise<any>> {
+  const processor = await import(`${PROCESSOR_PATH}/${processorFileName}`);
+  if (!processor.default) {
     console.warn('The imported module must have an export default statement');
     process.exit(1);
   }
-  return parser.default;
+  return processor.default;
 }
 
 async function main(mode: string) {
-  const parser = await getParser(`${mode}.parser.ts`);
+  const processor = await getProcessor(`${mode}.processor.ts`);
   const urls = await getUrls(`${mode}.urls.json`);
```

Comment:
you should rename `etimologias.json` as `etimologias.url.json`, otherwise it will never be loaded by this script.